front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -72,22 +72,8 @@ const StdcmConsist = ({ disabled = false }: StdcmConfigCardProps) => {
     }
   }, [towedRollingStock]);
 
-  const getLabel = (rs: LightRollingStockWithLiveries) => {
-    let res = '';
-    const { metadata, name } = rs;
-
-    if (metadata) {
-      const series = metadata.series ?? metadata.reference;
-      const subseries =
-        metadata.series && metadata.series !== metadata.subseries
-          ? metadata.subseries
-          : metadata.detail;
-      if (series) res += series;
-      if (subseries) res += series ? ` (${subseries})` : subseries;
-    }
-    if (name) res += ` -- ${name}`;
-    return res;
-  };
+  const getLabel = (rs: LightRollingStockWithLiveries) =>
+    `${rs.name} - ${rs.metadata?.series || rs.metadata?.reference || ''}`;
```

Comment:
If we don't have any metadata, we will have the '-', is it on purpose ?

Reply:
Nope, I pushed the fix